x-pack/plugins/lens/public/visualizations/metric/visualization.tsx

```diff
@@ -69,109 +75,206 @@ export interface MetricSuggestion extends Suggestion {
-          breakdownBy:
-            state.breakdownByAccessor && !state.collapseFn ? [state.breakdownByAccessor] : [],
-          subtitle: state.subtitle ? [state.subtitle] : [],
-          progressDirection: state.progressDirection ? [state.progressDirection] : [],
-          color: [state.color || getDefaultColor(!!state.maxAccessor)],
-          palette: state.palette?.params
-            ? [
-                paletteService
-                  .get(CUSTOM_PALETTE)
-                  .toExpression(computePaletteParams(state.palette.params as CustomPaletteParams)),
-              ]
-            : [],
-          maxCols: [state.maxCols ?? DEFAULT_MAX_COLUMNS],
-          minTiles: maxPossibleTiles ? [maxPossibleTiles] : [],
+        groupId: GROUP_ID.MAX,
+        dataTestSubj: 'lnsMetric_maxDimensionPanel',
+        groupLabel: i18n.translate('xpack.lens.metric.max', { defaultMessage: 'Maximum value' }),
+        paramEditorCustomProps: {
+          headingLabel: i18n.translate('xpack.lens.primaryMetric.headingLabel', {
+            defaultMessage: 'Value',
+          }),
         },
+        accessors: props.state.maxAccessor
+          ? [
+              {
+                columnId: props.state.maxAccessor,
+              },
+            ]
+          : [],
+        supportsMoreColumns: !props.state.maxAccessor,
+        filterOperations: isSupportedMetric,
+        enableDimensionEditor: true,
+        enableFormatSelector: false,
+        formatSelectorOptions: formatterOptions,
+        supportStaticValue: true,
+        prioritizedOperation: 'max',
+        groupTooltip: i18n.translate('xpack.lens.metric.maxTooltip', {
+          defaultMessage: 'If the maximum value is specified, the minimum value is fixed at zero.',
+        }),
+      },
+      {
+        groupId: GROUP_ID.BREAKDOWN_BY,
+        dataTestSubj: 'lnsMetric_breakdownByDimensionPanel',
+        groupLabel: i18n.translate('xpack.lens.metric.breakdownBy', {
+          defaultMessage: 'Break down by',
+        }),
+        accessors: props.state.breakdownByAccessor
+          ? [
+              {
+                columnId: props.state.breakdownByAccessor,
+                triggerIcon: props.state.collapseFn ? ('aggregate' as const) : undefined,
+              },
+            ]
+          : [],
+        supportsMoreColumns: !props.state.breakdownByAccessor,
+        filterOperations: isBucketed,
+        enableDimensionEditor: true,
+        enableFormatSelector: true,
+        formatSelectorOptions: formatterOptions,
       },
     ],
   };
 };
 
-export const metricLabel = i18n.translate('xpack.lens.metric.label', {
-  defaultMessage: 'Metric',
-});
-const metricGroupLabel = i18n.translate('xpack.lens.metric.groupLabel', {
-  defaultMessage: 'Goal and single value',
-});
+const getTrendlineLayerConfiguration = (
+  props: VisualizationConfigProps<MetricVisualizationState>
+): {
+  groups: VisualizationDimensionGroupConfig[];
+} => {
+  return {
+    groups: [
```

Comment:
@flash1293 just wanted to make sure I understood your hide-layer idea correctly.

You were suggesting that `visualization.getConfiguration` would be extended to return

```
{
  hidden: true,
  groups: [...],
}
```

Is that right?

Reply:
Yes, exactly